<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reader_window</class>
 <widget class="QMainWindow" name="reader_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>your library</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="library_page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>98</width>
       <height>28</height>
      </rect>
     </property>
     <attribute name="label">
      <string>&lt;books&gt;</string>
     </attribute>
     <widget class="QTabWidget" name="tab_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>681</width>
        <height>431</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="order_statuses_tab">
       <attribute name="title">
        <string>order statuses</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="create_order_tab">
       <attribute name="title">
        <string>create order</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="events_page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>98</width>
       <height>28</height>
      </rect>
     </property>
     <attribute name="label">
      <string>&lt;events&gt;</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="account_info_page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>681</width>
       <height>429</height>
      </rect>
     </property>
     <attribute name="label">
      <string>&lt;account info&gt;</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>phone:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>email:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phone_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="email_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>100</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>password:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update_info_button">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>update info</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
